keepassxc-browser/content/ui.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+// jQuery style wrapper for getElementById()
```

Comment:
```suggestion
// jQuery style wrapper for querySelector()
```

it's not only about IDs